CatProducer/plugins/RecoEventInfoProducer.cc

```diff
@@ -118,14 +124,23 @@ void RecoEventInfoProducer::produce(edm::Event& event, const edm::EventSetup& ev
       if ( hltPathsWithV.empty() ) continue;
       const std::string& trigName = hltPathsWithV[0];
       const unsigned int trigIndex = hltConfig_.triggerIndex(trigName);
+      HLTNameList.push_back(trigName);
+      const int psValue = hltConfig_.prescaleValue(event, eventSetup, trigName);
+      HLTPrescaleList.push_back(psValue);
+      
       if ( trigIndex < hltHandle->size() )
       {
-        if ( hltHandle->accept(trigIndex) ) { isPassed = true; break; }
-      }
+        //if ( hltHandle->accept(trigIndex) ) { isPassed = true; break; }
+        if ( hltHandle->accept(trigIndex) ) { isPassed = true; HLTListPassFail.push_back(1); break;} 
```

Comment:
This line have to be nested in the line 131 (originally in 121) for the safety.
```cpp
hltHandle->accept(trigIndex);
```
sometimes fails if the trigger path name is not present in the HLTResults.